core/src/main/java/org/lflang/target/property/DockerProperty.java

```diff
@@ -0,0 +1,144 @@
+package org.lflang.target.property;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import org.lflang.MessageReporter;
+import org.lflang.Target;
+import org.lflang.TargetProperty.DictionaryElement;
+import org.lflang.TargetPropertyConfig;
+import org.lflang.ast.ASTUtils;
+import org.lflang.lf.Element;
+import org.lflang.lf.KeyValuePair;
+import org.lflang.lf.KeyValuePairs;
+import org.lflang.lf.LfFactory;
+import org.lflang.target.property.DockerProperty.DockerOptions;
+import org.lflang.target.property.type.DictionaryType;
+import org.lflang.target.property.type.PrimitiveType;
+import org.lflang.target.property.type.TargetPropertyType;
+import org.lflang.target.property.type.UnionType;
+
+public class DockerProperty extends TargetPropertyConfig<DockerOptions> {
+
+  public DockerProperty() {
+    super(UnionType.DOCKER_UNION);
+  }
+
+  @Override
+  public DockerOptions initialValue() {
+    return new DockerOptions(false);
```

Comment:
I don't know why, but it seems like docker is enabled by default. When I run lfc on a C file, it prints:
```
lfc: info: Dockerfile written to /home/cmenard/projects/lf/lingua-franca/test/C/src-gen/Minimal/Dockerfile
lfc: info: #####################################
To build and run:
    pushd /home/cmenard/projects/lf/lingua-franca/test/C/src-gen/Minimal && docker compose up --build
To return to the current working directory afterwards:
    popd
#####################################
```

Reply:
Yep, still a bunch of bugs to squash...